### ISSUE TITLE
Add 'association/alliance' type to organizations

### ISSUE THREAD

**Issue by Simon-Cobb** [fnielsen/scholia#385]
To view the activities of a group comprised of two or more organisations such as the Ivy League (Q49113), Russell Group (Q590631), Utrecht Network (Q39129) etc. Same functionality as multiple organisations but from a single item.

**Comment by fnielsen** [comment]
How do individuals associate with groups. Is that only with P463? Or do we have other forms of connections?

**Comment by fnielsen** [comment]
As I understand you would like something like `scholia/organizational-group/Q39129`.

**Comment by Simon-Cobb** [comment]
Yes, that is correct. P463 is the most common connection with P527 used as the inverse property.  Thanks.

**Comment by Daniel-Mietchen** [comment]
Scholarly societies would be useful to have too.

**Comment by fnielsen** [comment]
Wouldn't scholarly societies be included as P463 (member of) is in the SPARQL, see, e.g., https://tools.wmflabs.org/scholia/organization/Q58532558

**Comment by Daniel-Mietchen** [comment]
@fnielsen yes, you're right.
